matrix-meetings-widget/src/components/meetings/ScheduleMeetingModal/ScheduleMeeting.tsx

```diff
@@ -459,6 +461,7 @@ export const ScheduleMeeting = ({
         <TextField
           fullWidth
           id={descriptionId}
+          inputProps={{ maxLength: 20000 }}
```

Comment:
Does it make sense to have some e2e test that could check that a meeting with these limits could be created without any error?

Reply:
I added two e2e tests for it.